src/Config_option.ml

```diff
@@ -62,8 +62,9 @@ module Make (C : CONFIG) = struct
   let deprecated_doc ~deprecated =
     if deprecated then " Warning: This option is deprecated." else ""
 
-  let generated_flag_doc ~allow_inline ~doc ~section ~deprecated =
-    Format.sprintf "%s%s%s" doc
+  let generated_flag_doc ~allow_inline ~doc ~section ~default ~deprecated =
+    let default = if default then "on" else "off" in
+    Format.sprintf "%s The default value is $(b,%s).%s%s" doc default
```

Comment:
I would write `This flag is set by default.` (unset)

Reply:
Looks better !